src/cryptsetup/cryptsetup-keyfile.c

```diff
@@ -1,110 +1,57 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
-#include <unistd.h>
-
 #include "cryptsetup-keyfile.h"
-#include "fd-util.h"
-#include "format-util.h"
-#include "memory-util.h"
+#include "fileio.h"
 #include "path-util.h"
-#include "stat-util.h"
 #include "strv.h"
 
-#define KEY_FILE_SIZE_MAX (16U*1024U*1024U) /* 16 MiB */
-
-int load_key_file(
+int search_key_file(
```

Comment:
The usual caveat about "search" vs. "search for" applies. `find_key_file` would be better.